packages/astro/src/@types/astro.ts

```diff
@@ -1142,10 +1147,12 @@ export interface AstroIntegration {
 		'astro:config:setup'?: (options: {
 			config: AstroConfig;
 			command: 'dev' | 'build';
+			isConfigReload: boolean;
```

Comment:
nit: maybe this should be `isRestart` as there could be more than just configs that trigger reloads/restart in the future?